styles.css

```diff
@@ -123,7 +122,7 @@ main{
 	position: absolute;
 	left: 75vw;
 	transform: translateX(-50%) translateY(-50%);
-	background-color: hsla(0, 0%, 100%, 0.7);
+	background-color: hsla(0, 0%, 100%, 1);
```

Comment:
Instead of changing the rule here where it affects other elements (the score and hints display) as well, could you add this rule to only the author display? (around line 135)